core/src/gl/typedMesh.h

```diff
@@ -37,37 +38,45 @@ class TypedMesh : public VboMesh {
      * after _byteOffset in the mesh vertex data memory
      */
     template<class A>
-    void updateAttribute(GLintptr _byteOffset, unsigned int _nVerts, const A& _newAttributeValue) {
+    void updateAttribute(std::array<int,2> _vertexRange, const A& _newAttributeValue, size_t _attribOffset = 0) {
```

Comment:
I may be missing something but any reason of using `std::array<int,2>` rather than `int _vertexRange[2]`? We could probably save an include.